pkgs/tools/security/gnupg/fix-libusb-include-path.patch

```diff
@@ -0,0 +1,11 @@
+--- a/configure
++++ b/configure
+@@ -8872,7 +8872,7 @@
+    { $as_echo "$as_me:${as_lineno-$LINENO}: checking libusb include dir" >&5
+ $as_echo_n "checking libusb include dir... " >&6; }
+    usb_incdir_found="no"
+-   for _incdir in "" "/usr/include/libusb-1.0" "/usr/local/include/libusb-1.0"; do
++   for _incdir in "" "/usr/include/libusb-1.0" "/usr/local/include/libusb-1.0" "$($PKG_CONFIG --variable=includedir libusb-1.0)/libusb-1.0"; do
```

Comment:
Why didn't you remove those impure paths from the list? Nix builds should never use those locations.